Source/WebKit/Shared/API/c/cairo/WKImageCairo.cpp

```diff
@@ -38,17 +38,18 @@
 
 cairo_surface_t* WKImageCreateCairoSurface(WKImageRef imageRef)
 {
-#if USE(SKIA)
-    // FIXME
-    return nullptr;
-#else
+#if USE(CAIRO)
     // We cannot pass a RefPtr through the API here, so we just leak the reference.
     return WebKit::toImpl(imageRef)->createCairoSurface().leakRef();
+#else
+    // FIXME
```

Comment:
By the way, gentle reminder that we have the `WebCore/NotImplemented.h` header that defines a `notImplemented()` macro that could be used there&mdash;no need to change it in this patch, as the `FIXME` was already there before, just a comment for future patches.